### ISSUE TITLE
Final Fantasy VII USA

### ISSUE THREAD

**Issue by davetansley** [MiSTer-devel/PSX_MiSTer#94]
**Issue:**
Attempting to save after killing Emerald Weapon causes E2 and core freeze.

**Reproduce:**

1. Load attached save state 
[Final Fantasy VII (USA) (Disc 3)_1.ss.zip](https://github.com/MiSTer-devel/PSX_MiSTer/files/8692825/Final.Fantasy.VII.USA.Disc.3._1.ss.zip)
2. Press triangle
3. Select save
4. Core will freeze

**Workaround:**
Save states?

**BIOS**
1001

**CD Image**
redump chd

**Core Version / Build date**
PSX_20220511.rbf

**Comment by Fuzzybane** [comment]
testing your save state now :)

**Comment by davetansley** [comment]
Just did a few more tests after talking through in discord:

1) Reconverted bin/cue with latest chdman - same E2 issue
2) Downloaded redump bin/cue again and reconverted - same E2 issue

Interestingly, I tried loading the bin/cue, restoring the state, then loading the chd directly after without rebooting and restoring the state. This didn't cause the E2.

**Comment by davetansley** [comment]
Did some further testing with another MiSTer I have - no issues. Others in the discord report they can't reproduce this either, so I suspect it's something to do with this particular hardware. I tried removing the PSX config files, but still the same problem.

Since it appears isolated, I'll close this ticket.

**Comment by davetansley** [comment]
Just to add some extra information for future readers who are bored and fancy a head scratcher... I tried copying the memory card to the working MiSTer and established a save after Emerald Weapon - just after the  save state. I copied this memory card back to the affected MiSTer, loaded the save and tried to access the save menu again - no E2.

So, to summarise:

1) Had a memory card save before Emerald - no E2 on save menu access
2) Had a save state after Emerald - E2 on save menu access
3) Had a memory card save created from the save state on another MiSTer - no E2 on save menu access on the affected MiSTer

I guess something got corrupted in memory that only affects this one MiSTer, which was then smoothed out by the memory card save on the other MiSTer. A weird one :)
